.github/workflows/ci-tests.yaml

```diff
@@ -57,7 +57,7 @@ jobs:
       - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.32
+          version: v1.45.2
```

Comment:
This is the only merge conflict I see and when accepting the incoming changes it passes

Reply:
I needed to upgrade to golangci-lint to make the tests pass. In the meanwhile also master branch has updated golangci-lint to v1.46.2 so I've removed this commit.

Reply:
Still, linting fails in CI workflow with:

```
Running [/home/runner/golangci-lint-1.46.2-linux-amd64/golangci-lint run --out-format=github-actions --timeout 5m] in [] ...
  panic: load embedded ruleguard rules: rules/rules.go:13: can't load fmt
  
  goroutine 1 [running]:
  github.com/go-critic/go-critic/checkers.init.22()
  	github.com/go-critic/go-critic@v0.6.3/checkers/embedded_rules.go:47 +0x4b4
  
  Error: golangci-lint exit with code 2
  Ran golangci-lint in 9[72](https://github.com/argoproj-labs/argocd-image-updater/actions/runs/4968280408/jobs/8890784534?pr=424#step:3:74)2ms
```
https://github.com/argoproj-labs/argocd-image-updater/actions/runs/4968280408/jobs/8890784534?pr=424

Going to update `golangci-lint` to most recent version.

Reply:
To fix `golangci-lint` issues I needed to:
* Update `golangci-lint` to 1.52.2
* Replace deprecated linters `deadcode`, `varcheck`, `structcheck` by the newer `unused` linter. See relevant `golangci-lint` PR at https://github.com/golangci/golangci-lint/issues/1841
* Run `goimports -w -local github.com/argoproj-labs/argocd-image-updater .` to fix
** Comment indentation
** Reorder imports